### DISCUSSION TITLE
Migrate python project to uv

### DISCUSSION THREAD

**Discussion by n-io** [InSyncWithFoo/ryecharm#40] [Q&A] [answered]
What is the best way to migrate an existing python project to uv? In particular, there are quite a few run configs that I would like to preserve. Is there a way to avoid deleting the pycharm python project and re-setting it up as a uv project?

**Comment by InSyncWithFoo** [comment, marked as answer]
I think this is a duplicate of #34. Does my comment there answer your question?

**Comment by n-io** [comment]
Thanks, I had found the configs but wasn't sure if anything else was required. The missing link for me that selecting an interpreter works the same way as in venv.
